spec/models/user_spec.rb

```diff
@@ -12,4 +13,16 @@
   describe 'validates uniqueness of email' do
     it { is_expected.to validate_uniqueness_of(:email) }
   end
+
+  describe 'valid email typed' do
+    it 'should be valid email typed like ' do
+      email = User.new(email: 'valid@email.com')
+      expect(email).to be_valid
+    end
+
+    it 'should not be valid email typed like ' do
+      email2 = User.new(email: 'invalid@invalid@.com')
+      expect(email2).to_not be_valid
```

Comment:
rename `email` to `user`
and check if user.errors contains the error that you are looking for

Reply:
ok